packages/animations/test/open_container_test.dart

```diff
@@ -0,0 +1,934 @@
+// Copyright 2019 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:animations/src/open_container.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/widgets.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  testWidgets('Container opens', (WidgetTester tester) async {
+    const ShapeBorder shape = RoundedRectangleBorder(
+      borderRadius: BorderRadius.all(Radius.circular(8.0)),
+    );
+    bool closedBuilderCalled = false;
+    bool openBuilderCalled = false;
+
+    await tester.pumpWidget(_boilerplate(
+      child: Center(
+        child: OpenContainer(
+          closedColor: Colors.green,
+          openColor: Colors.blue,
+          closedElevation: 4.0,
+          openElevation: 8.0,
+          closedShape: shape,
+          closedBuilder: (BuildContext context, VoidCallback _) {
+            closedBuilderCalled = true;
+            return const Text('Closed');
+          },
+          openBuilder: (BuildContext context, VoidCallback _) {
+            openBuilderCalled = true;
+            return const Text('Open');
+          },
+        ),
+      ),
+    ));
+
+    // Closed container has the expected properties.
+    final StatefulElement srcMaterialElement = tester.firstElement(
+      find.ancestor(
+        of: find.text('Closed'),
+        matching: find.byType(Material),
+      ),
+    );
+    final Material srcMaterial = srcMaterialElement.widget;
```

Comment:
Not sure why we don't use tester.widget<Material> here. Wouldn't this work?
```dart
final Material material = tester.widget<Material>(
  find.ancestor(
    of: find.text('Closed'),
      matching: find.byType(Material),
  ),
);
```

Reply:
We could do that here. But further down I am checking how the Widget configuration of this element is changing over time and it this makes the test more symmetric.